### ISSUE TITLE
Allow subtitle text to be provided as parameter (schema update)

### ISSUE THREAD

**Issue by jwoLondon** [vega/vega-lite#7408]
It is useful to be able to specify a subtitle's text with a parameter, just was we can with [main title text](https://vega.github.io/vega-lite/docs/title.html#props). Currently this does actually work, but the schema and documentation suggest it cannot. Can the schema be updated to allow subtitle [expression references](https://vega.github.io/vega-lite/docs/types.html#exprref)?

**Comment by domoritz** [comment]
Could do easily in https://github.com/vega/vega-lite/pull/7265

**Comment by domoritz** [comment]
Also part of https://github.com/vega/vega-lite/issues/7284.

**Comment by joelostblom** [comment]
In addition to supporting expressions in subtitles it would be great to support them for guide titles as well (I am thinking of axis and legends). It sounds like https://github.com/vega/vega-lite/pull/7438 might fix all of this, but thought I would leave a note here just in case. Once this is supported, it might be possible to write a macro to also support https://github.com/vega/vega-lite/issues/7365#issuecomment-1058909767 without as big changes as what originally suggested in that thread.

**Comment by domoritz** [comment]
Yep, I think https://github.com/vega/vega-lite/pull/7438 should add that.
